File: src/services/LibroService.js
import { AxiosInstance } from "../config/axios-config"

class LibroService {
    get() {
        return AxiosInstance.get("libro");
    }

    delete(id) {
        return AxiosInstance.delete(`libro/delete/${id}`);
    }

    create(libro) {
        return AxiosInstance.post('libro/create', libro);
    }

    update(id) {
        return AxiosInstance.put(`libro/update/${id}`);
    }
}

export default new LibroService();

File: APACHE NETBEANS(SERVICIOS)/VirtualMind/NETBEANS-main/BibliotecaVirtual/src/main/java/com/Biblioteca/Virtual/service/iface/LibroService.java
package com.Biblioteca.Virtual.service.iface;

import com.Biblioteca.Virtual.models.entity.Libro;
import java.util.List;

public interface LibroService {
    void create (Libro  libro);
    List<Libro> getAll();
    void update(int id,Libro libro);
    void delete (int id);
}
